lego/apps/users/tasks.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
+from math import ceil
+
+from django.conf import settings
+from django.db.models import Q
+from django.utils import timezone
+
+from structlog import get_logger
+
+from lego import celery_app
+from lego.apps.users.models import User
+from lego.apps.users.notifications import DeletedUserNotification, InactiveNotification
+from lego.utils.tasks import AbakusTask, send_email
+
+log = get_logger()
+
+MAX_INACTIVE_DAYS = 365
+MIN_INACTIVE_DAYS = MAX_INACTIVE_DAYS - 2 * 30
+MEDIAN_INACTIVE_DAYS = MAX_INACTIVE_DAYS - ceil(
+    (MAX_INACTIVE_DAYS - MIN_INACTIVE_DAYS) / 2
+)
+
+
+@celery_app.task(serializer="json", bind=True, base=AbakusTask)
+def send_inactive_reminder_mail(self, logger_context=None):
+
+    self.setup_logger(logger_context)
+
+    users_to_delete = User.objects.filter(
+        Q(last_login__lte=timezone.now() - timedelta(days=MAX_INACTIVE_DAYS))
+        & Q(inactive_notified_counter__gte=4)
+    )
+
+    list_usernames_to_delete = list(map(lambda u: u.username, users_to_delete))
+    num_users_to_delete = len(users_to_delete)
+    if len(list_usernames_to_delete) != num_users_to_delete:
```

Comment:
Will this statement ever be `False`? Would it make more sense to compare `len(set(list_usernames_to_delete))` and `num_users_to_delete)`?